code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -370,6 +370,17 @@
 				if(SA.stat || in_faction(SA)) //don't target if dead or in faction
 					continue
 				targets += SA
+			if(issilicon(A))
+				var/mob/living/silicon/sillycone = A
+				if(sillycone.stat || in_faction(sillycone))
+					continue
+
+				if(iscyborg(sillycone))
+					var/mob/living/silicon/robot/sillyconerobot = A
+					if(faction == "syndicate" && sillyconerobot.emagged == 1)

```

Comment:
Did you mean this to be an OR statement? I understand line 375 to avoid shooting silicons in the turret's faction or dead ones. Is this line here intended to make **Nanotrasen** turrets never shoot Syndicate or emagged cyborgs?

Reply:
Nanotrasen turrets would not shoot at emagged borgs unless emagging also changes the faction of the borg (which isn’t how it currently works).

Reply:
The line is to make syndicate turrets not shoot emagged borgs even when the borg isn’t in their faction

Reply:
But you use an `AND` statement here, meaning that the borg must both have the Syndicate faction AND be emagged in order to be ignored by the turret. I might be an idiot, in which case just disregard me.

Reply:
this is the turret's faction, not the borg's